python/cugraph/link_prediction/wjaccard.py

```diff
@@ -69,7 +69,7 @@ def jaccard_w(input_graph, weights, vertex_pair=None):
     >>> weights = cudf.Series(numpy.ones(
     >>>     max(sources.max(),destinations.max())+1, dtype=numpy.float32))
```

Comment:
67-69 not needed

Reply:
The weights are needed "df = cugraph.jaccard_w(G, weights)"

Reply:
fixed